examples/tensorflow-privacy/README.md

```diff
@@ -27,40 +29,35 @@ This will create a new directory called `tensorflow-privacy` containing the foll
 -- README.md
 ```
 
-### Installing dependencies
-
-Project dependencies are defined in `pyproject.toml`. Install them with:
-
 ```shell
-pip install .
+tensorflow-privacy
+├── tf_privacy
+│   ├── client_app.py   # Defines your ClientApp
+│   ├── server_app.py   # Defines your ServerApp
+│   └── task.py         # Defines your model, training, and data loading
+├── pyproject.toml      # Project metadata like dependencies and configs
+└── README.md
 ```
 
-## Run Flower with tensorflow-privacy and TensorFlow
+### Note on the Code (Privacy Hyperparameters)
 
-### 1. Start the long-running Flower server (SuperLink)
-
-```bash
-flower-superlink --insecure
-```
+Please note that, at the current state, users cannot set `NodeConfig` for simulated `ClientApp`s. For this reason, the hyperparameter `noise_multiplier` is set in the `client_fn` method based on a condition check on `partition_id`. This will be modified in a future version of Flower to allow users to set `NodeConfig` for simulated `ClientApp`s.
 
-### 2. Start the long-running Flower clients (SuperNodes)
+### Install dependencies and project
 
-Start 2 Flower `SuperNodes` in 2 separate terminal windows, using:
+Install the dependencies defined in `pyproject.toml` as well as the `tf_privacy` package.
 
-```bash
-flower-client-app client:appA --insecure
-```
-
-```bash
-flower-client-app client:appB --insecure
+```shell
+# From a new python environment, run:
+pip install -e .
 ```
 
-tensorflow-privacy hyperparameters can be passed for each client in `ClientApp` instantiation (in `client.py`). In this example, `noise_multiplier=1.5` and `noise_multiplier=1` are used for the first and second client respectively.
+## Run the project
 
-### 3. Run the Flower App
+You can run your Flower project in both _simulation_ and _deployment_ mode without making changes to the code. If you are starting with Flower, we recommend you using the _simulation_ mode as it requires fewer components to be launched manually. By default, `flwr run` will make use of the Simulation Engine.
 
-With both the long-running server (SuperLink) and two clients (SuperNode) up and running, we can now run the actual Flower App:
+### Run with the Simulation Engine
 
 ```bash
-flower-server-app server:app --insecure
+flwr run .
```

Comment:
Would it make sense to extend slightly the readme to just show how to override the `--run-config`?